x-pack/plugins/maps/common/execution_context.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { APP_ID } from './constants';
+
+export function makeExecutionContext(type: string, id: string, description: string, url: string) {
```

Comment:
I would recommend removing this method. It does not provide any value and adds a layer of abstraction that is not needed and just makes following the logic a single hop harder. Instead, just create the context object directly in makePublicExecutionContext.